bundle-workflow/src/test_workflow/put_file_into_S3.py

```diff
@@ -0,0 +1,361 @@
+# !/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+import argparse
+import json
+import os
+
+from src.read_write_s3 import S3Bucket
+
+
+class ConvertWrite:
+    """
+    result_file- the file where the console output will be stored
+    assume_role- Amazon Resource Name (ARN) of the role that you want to assume
+    session- Name of the session
+    bucket_name- Name of the Bucket
+    object_path- Path in the Bucket where you want to store the file
+    file_path- File path where the console output result_file is present
+    """
+    def parse_arguments(self):
+        parser = argparse.ArgumentParser()
+        parser.add_argument("-r", "--result_file", help="Result file", required=True)
+        parser.add_argument("-a", "--assume_role", nargs="?", default="os.getenv('OPENSEARCH_TESTING_ROLE')", help="Assume role")
+        parser.add_argument("-s", "--session", nargs="?", default="get-files-session", help="Session name")
+        parser.add_argument("-b", "--bucket_name", help="Bucket name", required=True)
+        parser.add_argument("-o", "--object_path", help="ObjectPath in Bucket", required=True)
+        parser.add_argument("-f", "--file_path", help="Filepath of the result file", required=True)
+
+        args = parser.parse_args()
+
+        return args
+
+    def conversion_to_html(self, args):
+        with open(args.file_path + args.result_file) as json_data:
+            try:
+                data = json.load(json_data)
+                testExecutionId = data['testExecutionId']
+                filename = "%s.html" % testExecutionId
+                values = {1: 'opCount', 2: 'opErrorCount', 3: 'opErrorRate', 4: 'p0', 5: 'p50', 6: 'p100', 7: 'p50',
```

Comment:
Hardcoding the model characteristics into these objects strongly couples their ordering.  If there is an update to add additional fields this will be terrible to maintain.  There are several paradigms for this in software development such as [Model View Controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) that create contracts between the components to ease their reuse.